### PR TITLE
made keyword arguments explicit

### PR DESCRIPTION
**Pull request by thisfred**
This passes through the keyword arguments explicitly so that typo's don't lead to hard to debug behavior but rather to syntax errors.

In one instance a `create` keyword argument was passed, which I assume was meant to be either `create_client` or `create_classpath`, but i'm not sure which, so I picked one, and added a comment. Please let me know either way, and I'll remove the comment and/or fix the usage.

### CODE REVIEW

File: ensime_shared/ensime.py [hunk 1]
```diff
@@ -857,7 +861,9 @@ def client_keys(self):
 
     def client_status(self, config_path):
         """Get the client status of a given project."""
-        c = self.client_for(config_path, create=True)
+        # XXX: `create` wasn't used, should this be `create_client` or
+        # `create_classpath`?
+        c = self.client_for(config_path, create_client=True)
```
Comment (thisfred) [inline ensime_shared/ensime.py]: this is the change I'm not sure about

Comment (jvican) [inline ensime_shared/ensime.py]: This is a minor change that I did in my last PR. Could you please remove this change?

**Comment by jvican** [comment]
I like the idea of passing explicit parameters but if the number of them starts to increase we may be better of going back to this solution. Please, remove that change I pointed out and rebase on top of the last PR. Then it'll get merged :+1:

File: ensime_shared/ensime.py (hunk 1 above)
Comment (thisfred) [inline ensime_shared/ensime.py]: so remove the create parameter altogether?

**Comment by jvican** [comment]
Are you ready for merge @thisfred ?

**Comment by thisfred** [comment]
yep, this one's ready too, but needs #158 to go in first

**Comment by jvican** [comment]
Oops, actually you'll need to rebase this on top of the other PR so that it can get merged.

**Comment by thisfred** [comment]
I redid the branch with just the one commit cherry picked.

**Comment by jvican** [comment]
Thanks @thisfred!

**Pull request merged**